nb-adapters/adapter-gcp-spanner/src/main/resources/gcp-spanner.md

```diff
@@ -0,0 +1,30 @@
+# Google Spanner driver adapter
```

Comment:
The name of this file and the adapter name should match exactly. Can we simplify them both to just `spanner` or at least `gcp_spanner` in both cases?

Reply:
renamed in both places to simply 'spanner'